### PR TITLE
Update metallb to 0.13.6

### PR DESCRIPTION
**Pull request by jacob-delgado**

### CODE REVIEW

File: files/common/scripts/metallb.yaml [hunk 1]
```diff
@@ -232,168 +1458,349 @@ roleRef:
...
+            fieldRef:
+              fieldPath: status.podIP
+        - name: METALLB_ML_LABELS
+          value: app=metallb,component=speaker
+        - name: METALLB_ML_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              key: secretkey
+              name: memberlist
+        image: quay.io/metallb/speaker:v0.13.6
```
Comment (howardjohn) [inline files/common/scripts/metallb.yaml:1612]: can we use anything other than quay? They are very unreliable

Comment (jacob-delgado) [reply files/common/scripts/metallb.yaml:1612]: Metallb has moved from dockerhub to quay.io. See https://hub.docker.com/r/metallb/speaker/tags as it hasn't been updated for 4 months

If you can move this to gcr.io I can update the image tags

Comment (howardjohn) [reply files/common/scripts/metallb.yaml:1612]: copied to `gcr.io/istio-testing/metallb/{speaker,controller}:v0.13.6`

**Comment by jacob-delgado** [comment]
Closing due to https://github.com/metallb/metallb/issues/1473

Will re-open a new PR for 0.12

**Comment by howardjohn** [comment]
ill copy 0.12.1 as well then

**Pull request closed without merging**